src/lib/project_workspace.ts

```diff
@@ -24,8 +24,14 @@ export class ProjectWorkspace {
    */
   pathToJest: string;
 
-  constructor(rootPath: string, pathToJest: string) {
+  /**
+   * The path to configuration file
```

Comment:
would love to write the semantics of how this is used too please ( e.g. "treat as ignored if zero length" )